charts/app-config/values-integration.yaml

```diff
@@ -206,6 +206,147 @@ govukApplications:
           secretKeyRef:
             name: rails-secret-key-base
             key: SECRET_KEY_BASE
+- name: content-data-admin
+  helmValues:
+    workerEnabled: true
+    workerReplicaCount: 1
+    extraEnv:
+      - name: CONTENT_DATA_API_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-content-data-admin-content-data-api
+            key: bearer_token
+      - name: GOOGLE_TAG_MANAGER_ID
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-google
+            key: manager-id
+      - name: GOOGLE_TAG_MANAGER_PREVIEW
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-google
+            key: manager-preview
+      - name: GOOGLE_TAG_MANAGER_AUTH
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-google
+            key: manager-auth
+      - name: SECRET_KEY_BASE
+        valueFrom:
+          secretKeyRef:
+            name: rails-secret-key-base
+            key: SECRET_KEY_BASE
+      - name: GDS_SSO_OAUTH_ID
+        valueFrom:
+          secretKeyRef:
+            name: signon-app-content-data-admin
+            key: oauth_id
+      - name: GDS_SSO_OAUTH_SECRET
+        valueFrom:
+          secretKeyRef:
+            name: signon-app-content-data-admin
+            key: oauth_secret
+      - name: AWS_ACCESS_KEY_ID
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-aws
+            key: access_key
+      - name: AWS_SECRET_ACCESS_KEY
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-aws
+            key: secret_key
+      - name: AWS_REGION
+        value: eu-west-1
+      - name: AWS_CSV_EXPORT_BUCKET_NAME
+        value: govuk-integration-content-data-csvs
+      - name: DATABASE_URL
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-postgres
+            key: DATABASE_URL
+      - name: GOVUK_NOTIFY_API_KEY
+        valueFrom:
+          secretKeyRef:
+            name: content-data-admin-notify
+            key: GOVUK_NOTIFY_API_KEY
+      - name: GOVUK_NOTIFY_TEMPLATE_ID
+        value: 759acac6-da53-4a19-b591-b7538c7c39de
+      - name: REDIS_URL
+        value: redis://shared-redis-govuk.eks.integration.govuk-internal.digital
+- name: content-data-api
+  helmValues:
+    dbMigrationEnabled: true
+    workerEnabled: true
+    workerReplicaCount: 1
+    extraEnv:
+      - name: GOOGLE_ANALYTICS_GOVUK_VIEW_ID
+        valueFrom:
+          secretKeyRef:
+            name: content-data-api-google
```

Comment:
Similarly with this one — it would make sense for it to be `google-analytics` rather than just `google`. And probably just `view-id` instead of `govuk-view-id`, unless there's more than one view ID? 

(Also, should the view ID be part of the secret? I suspect not, as it's not itself a secret and it wouldn't be needed in order to rotate the key.)